internal/pkg/config/config.go

```diff
@@ -29,10 +29,11 @@ var (
 
 // InjectionConfig is a specific instance of a injected config, for a given annotation
 type InjectionConfig struct {
-	Name        string             `json:"name"`
-	Containers  []corev1.Container `json:"containers"`
-	Volumes     []corev1.Volume    `json:"volumes"`
-	Environment []corev1.EnvVar    `json:"env"`
+	Name         string               `json:"name"`
+	Containers   []corev1.Container   `json:"containers"`
+	Volumes      []corev1.Volume      `json:"volumes"`
+	Environment  []corev1.EnvVar      `json:"env"`
+	VolumeMounts []corev1.VolumeMount `json:"volumeMounts"`
```

Comment:
My only comment on the PR would have been the use of `corev1.VolumeMount` As singular vs `VolumeMounts` In most other places. There is some mix between singular and plular throughout that is not clear to me why it needs to be different.

Reply:
Well, `corev1.VolumeMount` refers to the Kubernetes specification of a single volumeMount in a container object. Now that we want to have more than one volumeMount in our config, I named it `volumeMounts` which is referring to a list of `volumeMounts`